Dockerfile

```diff
@@ -1,11 +1,21 @@
-FROM rtfpessoa/ubuntu-jdk8:latest
-MAINTAINER Rodrigo Fernandes <rodrigo@codacy.com>
+FROM codacy/docker-java:0.0.1
 
-RUN locale-gen en_US.UTF-8
+LABEL maintainer="Rodrigo Fernandes <rodrigo@codacy.com>"
 
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US.UTF-8
-ENV LC_ALL en_US.UTF-8
+RUN \
+  apt-get -y update && \
+  add-apt-repository -y ppa:git-core/ppa && \
+  apt-get -y update && \
+  apt-get -y install curl wget unzip vim && \
```

Comment:
these are not needed

Reply:
Do you mind if I ask,,how do you know that?

I prefer to not do it myself since it is quite hard to track.

Reply:
That's correct it's hard to track, but we can start from a clean image and re-add if needed:
- `vim` -> pretty sure it's not used
- `curl` | `wget` -> used in scripts in doplicated and ansible AFAIK I'm fine in leaving one of the two
- `unzip` -> used in ansible

cc. @paulopontesm

Reply:
I am updating the base. I don't think it makes sense to break stuff in the middle.
We can do that later after the usages are removed from those places.

Reply:
I removed vim, but kept the others for now. It's just a couple of MBs anyway.